css/styles.css

```diff
@@ -552,4 +555,125 @@ textarea {
     margin-top: 50px;
     padding: 20px;
   }
-}
\ No newline at end of file
+
+  .form-inner input {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    color: #6b778c;
+  }
+
+  ::-webkit-input-placeholder {
+    text-align: left;
+  }
+  
+  :-moz-placeholder {
+    text-align: left;
+  }
+
+  .form-inner textarea {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    align-items: start;
+    color: #091e42;
+  }
+
+  .form-inner input,
+  .form-inner textarea {
+    box-sizing: border-box;
+    background: #fff;
+    border: 1px solid #c8cfcb;
+    border-radius: 4px;
+    text-indent: 0;
+  }
+}
+
+form {
+  position: relative;
+  width: 100%;
+  border-radius: 1px;
+  background: #fff;
+}
```

Comment:
- [ Optional] I recommend changing the background color for the form from white to ` rgba(237, 237, 240, 1)` to match the design.

Reply:
Done...thanks for the suggestion